src/Pages/Root.tsx

```diff
@@ -51,14 +52,19 @@ export default function RootPage() {
   }
 
   const isGlobal = loggedOut || tab.value === RootTab.Global.value;
-  const timelineSubect: TimelineSubject = isGlobal
-    ? { type: "global", items: [], discriminator: "all" }
-    : { type: "pubkey", items: follows, discriminator: "follows" };
+  const timelineSubect: TimelineSubject = (() => {
+    if (isGlobal) {
+      return { type: "global", items: [], discriminator: "all" };
+    }
+    if (tab.value >= 3) {
+      return { type: "hashtag", items: [tab.text.slice(1)], discriminator: "all" };
```

Comment:
`discriminator` should be the tag or else it will pull in the cached values for different tags

subId will be `timeline:hashtag:all` which is the same no matter what items you pass in

Reply:
fixed.